Makefile

```
@@ -30,7 +30,7 @@ clean:
 
 deps:
 	$(call title, "Installing dependencies")
-	dep ensure -v
+	go mod vendor
```

```suggestion
	go mod vendor -v
```

Do we want to enable verbose mode?:)